portfolio/src/main/webapp/script.js

```diff
@@ -63,24 +63,21 @@ async function sendAndGet() {
   });
 }
 
-function getCommentsHome() {
-  fetch('/data').then(response => response.json()).then((msgs) => {
-    const statsListElement = document.getElementById('home-comments-container');
-    statsListElement.innerHTML = '';
-    msgs.forEach((msg) => {
-      statsListElement.appendChild(
-        createListElement(msg.nickname + ': ' + msg.message));
-      statsListElement.appendChild(
-        createImgElement(msg.image));
-    })
-  });  
-}
-
+// Makes a Get request and loads Posts to the Blog page on body load
+// Also loads 5 display-only posts to be displayed on the home page.
 async function loadPosts(){
   fetch('/data').then(response => response.json()).then((msgs) => {
     const statsListElement = document.getElementById('posts-list');
+    const homeListElement = document.getElementById('home-comments-container');
+    var i = 1;
     msgs.forEach((msg) => {
       statsListElement.appendChild(createListElement(msg));
+      console.log('value :' + i);
+      if (i > 5) {
```

Comment:
I'd recommend defining the "5" as a constant on the top of this file like:

const MAX_POST_NUMBER = 5;

Then use the variable here.